dev/tools/localization/gen_l10n.dart

```diff
@@ -169,7 +247,23 @@ List<String> genIntlMethodArgs(Map<String, dynamic> bundle, String key) {
     if (attributesMap.containsKey('placeholders')) {
       final Map<String, dynamic> placeholders = attributesMap['placeholders'] as Map<String, dynamic>;
       if (placeholders.isNotEmpty) {
-        final String args = placeholders.keys.join(', ');
+        final List<String> argumentList = <String>[];
+        for (String placeholder in placeholders.keys) {
+          final dynamic value = placeholders[placeholder];
+          if (_isDateParameter(value)) {
```

Comment:
Best to find a way to include the error reporting only once. Maybe:
```
   if (_isDateParameter(value) && _DateParameterIsValid(value)) {
     argumentList.add('${placeholder}String');
   } else ...
```
Where _DateParameterIsValid can throw an exception